acceptance-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+python manage.py migrate
+python manage.py runscript initial_data
+python manage.py runscript test_data
+lsof -i tcp:9500| awk 'NR!=1 {print $2}' | xargs kill
+python manage.py runserver 9500 < /dev/null &
```

Comment:
Because this starts the server using `&`, after `tox` finishes, this server is still left running in the background. Can this script (or how `tox` invokes it) be changed to make sure it gets shutdown?